test/helpers/kubectl.go

```diff
@@ -321,58 +300,44 @@ func (kub *Kubectl) CiliumImportPolicy(namespace string, filepath string, timeou
 		}
 		return false
 	}
-	err = WithTimeout(body, "Can't import policy correctly due timeout", &TimeoutConfig{Timeout: timeout})
+	err = WithTimeout(
+		body,
+		"cannot import policy correctly; command timed out",
+		&TimeoutConfig{Timeout: timeout})
 	if err != nil {
 		return "", err
 	}
 	return "", nil
 }
 
-//CiliumPolicyDeleteAll delete all policies on cilium pods
-func (kub *Kubectl) CiliumPolicyDeleteAll(namespace string) error {
-	var revision int
-	pods, err := kub.GetCiliumPods(namespace)
-	if err != nil {
-		return err
-	}
-
-	for _, v := range pods {
-		revi, err := kub.CiliumPolicyRevision(v)
-		if err != nil {
-			return err
-		}
-
-		if revi > revision {
-			revision = revi
-		}
-	}
-	podA := pods[0]
-	_, err = kub.Exec(namespace, podA, "cilium policy delete --all")
-	if err != nil {
-		return err
-	}
-	newRevision, err := kub.CiliumPolicyRevision(podA)
-	if err != nil {
-		return err
-	}
+//CiliumReport report the cilium pod to the log and apppend the logs for the
```

Comment:
retrieves the logs for the specified pod, and retrieves the outputs of all commands ran in the cilium pod. Returns an error if running any of the commands in commands fails.